app/views/posts/show.html.erb

```diff
@@ -23,4 +23,5 @@
         </li>
       <% end %>
     </ul>
-  <% end %>
\ No newline at end of file
+  <% end %>
+  
```

Comment:
- [ ] [OPTIONAL AND IMPORTANT] For a nice user interface, kindly add buttons for a user to like and add comments, currently, I have to manually change the route to create comments and likes, this is no user friendly at all 
![Screenshot from 2023-10-05 23-18-34](https://github.com/meryemsanem/blog-app/assets/66097371/581d8897-c71b-4532-af0e-cc2ced258179)

Reply:
I will work on it. 
Thank you ☺️🙏🏻 @VICTOR-CHUKWU